databases/hello-app-cloud-spanner/k8s/deployment.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello-app-cloud-spanner
+  labels:
+    app: hello-app-cloud-spanner
+spec:
+  selector:
+    matchLabels:
+      app: hello-app-cloud-spanner
+  template:
+    metadata:
+      labels:
+        app: hello-app-cloud-spanner
+    spec:
+      containers:
+      - name: hello-app-cloud-spanner
+        image: us-docker.pkg.dev/google-samples/containers/gke/hello-app-cloud-spanner
```

Comment:
This might be wrong

Reply:
Sorry, I thought this might be wrong, because we use `us-docker...` instead of something like `us-west1-docker` (missing the compute region). Maybe we don't need it?

Reply:
No, that's how they're formatted, e.g.: `us-docker.pkg.dev/google-samples/containers/gke/hello-app`

Reply:
Got it, thanks